app/Domain/Goalcanvas/Templates/canvasDialog.blade.php

```diff
@@ -0,0 +1,182 @@
+
+    <h1><i class="fas <?= $canvasTypes[$canvasItem['box']]['icon']; ?>"></i> <?= $canvasTypes[$canvasItem['box']]['title']; ?></h1>
+
+    {{ $tpl->displayNotification() }}
+
+    <form class="formModal" method="post" action="{{ BASE_URL."/$canvasName/canvas/editCanvasItem/$id" }}">
+
+        <input type="hidden" value="{{ $currentCanvas }}" name="canvasId">
+        <input type="hidden" value="{{ $canvasItem['box'] }}" name="box" id="box">
+        <input type="hidden" value="{{ $id }}" name="itemId" id="itemId">
+        <input type="hidden" name="milestoneId" value="{{ $canvasItem['milestoneId'] ?? '' }}">
+        <input type="hidden" name="changeItem" value="1">
+
+        <div class="col-md-8">
+            <label>{{ __("label.what_is_your_goal") }}</label>
+            <input type="text" name="title" value="{{ $canvasItem['title'] }}" style="width:100%"><br>
+
+            @if (!empty($relatesLabels))
+            <label>{{ __("label.relates") }}</label>
+            <select name="relates" style="width: 50%" id="relatesCanvas">
+            </select><br>
+            @else
+            <input type="hidden" name="relates" value="{{ $canvasItem['relates'] ?? array_key_first($hiddenRelatesLabels) }}">
+            @endif
+            <br>
+            <h4 class="widgettitle title-light" style="margin-bottom:0px;"><i class="fa-solid fa-ranking-star"></i> {{ __("Metrics") }}</h4>
+
+            @php
+
+            $tpl->dispatchTplEvent('beforeMeasureGoalContainer', $canvasItem); @endphp
+            <div id="measureGoalContainer">
+                <label>How will you measure this objective. What metric will you be using.</label>
+                <input type="text" name="description" value="{{ $canvasItem['description'] }}" style="width:100%"><br>
+            </div>
+
+            <div class="row">
+                <div class="col-md-3">
+                    <label>Starting Value</label>
+                    <input type="number" step="0.01" name="startValue" value="{{ $canvasItem['startValue'] }}" style="width:105px">
+                </div>
+                <div class="col-md-3">
+                    <label>Current Value</label>
+                    <input type="number" step="0.01" name="currentValue" id="currentValueField" value="{{ $canvasItem['currentValue'] }}" @if ($canvasItem['setting']=='linkAndReport' ) readonly data-tippy-content="Current value calculated from child goals" @endif style="width:105px">
+                </div>
+                <div class="col-md-3">
+                    <label>Goal Value</label>
+                    <input type="number" step="0.01" name="endValue" value="{{ $canvasItem['endValue'] }}" style="width:105px">
+                </div>
+                <div class="col-md-3">
+                    <label>Type</label>
+                    <select name="metricType">
+                        <option value="number" @if ($canvasItem['metricType']=='number' ) selected @endif>Number</option>
```

Comment:
use language file. We should have existing labels for some of these. Please search translation file first